haiku-apps/voxophone/voxophone-1.0.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="An online radio player"
+DESCRIPTION="Voxophone is an online radio player application for Haiku."
+HOMEPAGE="https://github.com/aarroyoc/Voxophone"
+COPYRIGHT="2015 Adrian Arroyo Calle"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="67514047957bb8a186673ccc6fa71d2d0a1e1728"
+SOURCE_URI="https://github.com/aarroyoc/Voxophone/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bc98a70248057aa482f038e3689876dc0632aca4aa65d2417d9f692c86b486b5"
+SOURCE_FILENAME="voxophone-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Voxophone-$srcGitRev"
+PATCHES="voxophone-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	voxophone = $portVersion
+	app:Voxophone = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++
+	cmd:make
+	"
+	
```

Comment:
```diff
-	
+
```
(there's a tab)

Reply:
Whoops!
(I think I told you I'm terrible with whitespaces)
If only Github had an option to show all the whitespaces :(